docs/documentation/development/bindings/thing-definition.md

```diff
@@ -310,15 +310,15 @@ Having this property identified per binding it could be used as the `representat
 
 The `representation property` will be defined in the thing type XML: 
 
-```xml 
- <bridge-type id="thingTypeId">
+```xml
```

Comment:
Well, this was a copy/paste from the hue bridge thing type. Therefore it was bridge type. Not sure, if this should be changed.

Reply:
Of course the closing tag should also be bridge type.

Reply:
The `representation property` is valid for both. It can be used to identify bridge types as well as thing types. So it doesn't matter which we use in this example. Imho the thing type fits better, because this page is about things. ;-)